notebooks/explainable-ai-3-map-interpretation/imagewoof_use_cases.json

```diff
@@ -0,0 +1,40 @@
+{
+    "True_positive_high_confidence": {
```

Comment:
is it possible do not store this data in repository? In some cases notebooks users may do not clone repository and download only one notebook for running it instead (or open it with colab). Having extra files requires to be download in this case